sympy/core/evalf.py

```diff
@@ -1372,50 +1372,51 @@ class EvalfMixin(object):
 
     def evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None, verbose=False):
         """
-        Evaluate the given formula to an accuracy of n digits.
-        Optional keyword arguments:
+        Evaluate the given formula to an accuracy of *n* digits.
 
-            subs=<dict>
-                Substitute numerical values for symbols, e.g.
-                subs={x:3, y:1+pi}. The substitutions must be given as a
-                dictionary.
+        Parameters
+        ==========
 
-            maxn=<integer>
-                Allow a maximum temporary working precision of maxn digits
-                (default=100)
+        subs : dict, optional
+            Substitute numerical values for symbols, e.g.
+            ``subs={x:3, y:1+pi}``. The substitutions must be given as a
+            dictionary.
 
-            chop=<bool>
-                Replace tiny real or imaginary parts in subresults
-                by exact zeros (default=False)
+        maxn : int, optional
+            Allow a maximum temporary working precision of maxn digits.
 
-            strict=<bool>
-                Raise PrecisionExhausted if any subresult fails to evaluate
-                to full accuracy, given the available maxprec
-                (default=False)
+        chop : bool, optional
```

Comment:
This might be a good time to clarify:
```python
>>> N(1e-4, chop=True)
0.000100000000000000
>>> N(eq,chop=1e-5)
0.000100000000000000
>>> N(eq,chop=1e-4)
0
```
(when True the chop value defaults to standard precision, otherwise the chop value is used to determine the magnitude of "small" for purposes of chopping)